lib/std/fs.zig

```diff
@@ -501,7 +501,7 @@ pub const Dir = struct {
         },
         .linux => struct {
             dir: Dir,
-            buf: [8192]u8, // TODO align(@alignOf(os.dirent64)),
+            buf: [8192]u8 align(if (builtin.os.tag != .linux) 1 else @alignOf(os.linux.dirent64)),
```

Comment:
`builtin.os.tag != .linux` will always be false here.

Reply:
Yes, but it's there to prevent compile error from `os.linux.dirent64` definition not being imported when compiling for other OSes.

Reply:
Ok, then a comment is a good idea 👍

Reply:
Done :+1: